config/cams_schema.json

```diff
@@ -59,6 +59,14 @@
             "name": "iNatURL",
             "type": "String",
             "length": 256
+        },
+        "Reported Longitude":{
```

Comment:
I'm wondering if `External Longitude` might be a better name for this? Let's review when we run it past Keith.